src/sorcha/ephemeris/simulation_setup.py

```diff
@@ -46,9 +43,9 @@ def create_assist_ephemeris(args) -> tuple:
     """
     pplogger = logging.getLogger(__name__)
 
-    retriever = make_retriever(args.ar_data_file_path)
-    planets_file_path = retriever.fetch(JPL_PLANETS)
-    small_bodies_file_path = retriever.fetch(JPL_SMALL_BODIES)
+    retriever = make_retriever(sconfigs, args.ar_data_file_path)
```

Comment:
Yeah, that makes sense. One way to do this would be to have an independent class or dictionary or something has only the ephemeris stuff, so that it can also be created independently in bootstrap. Or if `sconfigs` doesn't need to have everything set up, it'd accomplish the same thing (with no need to change the code)

Reply:
Apparently I commented and did not finish the review so no one could see me comments. Thanks @Little-Ryugu  for pointing that out. - there is a smaller ephemeris auxiliary file class instance that is made within the sconfigs class so we should just be able to pass the ephemeris auxiliary file class here